pom.xml

```diff
@@ -90,5 +90,11 @@
             <version>1.2</version>
         </dependency>
 
+        <dependency>
```

Comment:
If you notice, I always add a comment line atop each dependency in order to know what it is for and we need it. It's good to avoid the proliferation of unnecessary dependencies. It's also good for others learning to write Jenkins plugins. Please add the corresponding comment line above.